src/runtime_src/core/common/api/bo_int.h

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2024 Advanced Micro Devices, Inc. All rights reserved.
+//
+#ifndef _XRT_COMMON_BO_INT_H_
+#define _XRT_COMMON_BO_INT_H_
+
+// This file defines implementation extensions to the XRT BO APIs.
+#include "core/include/experimental/xrt_bo.h"
```

Comment:
```suggestion
#include "core/include/xrt/xrt_bo.h"
```

Reply:
Good to know that the flags are not cached. OK, I can make it a no-op one.